app/main.py

```diff
@@ -1,8 +1,45 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str
+                 ):
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: int, clean_power: int, average_rating: float, count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = round(average_rating, 1)
+        self.count_of_ratings = count_of_ratings
+
+
+
+    def calculate_washing_price(self, car: Car) -> float:
+        price = (car.comfort_class * (self.clean_power - car.clean_mark) *
+                self.average_rating / self.distance_from_city_center)
+        return round(price, 1)
+
+
+    def wash_single_car(self, car: Car):
+        if car.clean_mark < self.clean_power:
+            car.clean_mark = self.clean_power
+
+
+    def serve_cars(self, cars: list[Car]) -> float:
```

Comment:
The `serve_cars` method assumes that the `cars` list contains only `Car` objects. Ensure that this assumption is valid or add type checking to handle potential errors.